components/people/directory/directory.tsx

```diff
@@ -42,13 +42,13 @@ const Directory = () => {
     useEffect(() => {
         setPage(0);
         setIsNextPageLoading(false);
-        dispatch(fetchProfiles(0, 15));
+        dispatch(fetchProfiles(0, 60));
```

Comment:
nit: In next PR's we could start handling all these values as constants (width, height, items, page size, etc).